misc/FFNx.lighting.frag

```diff
@@ -104,9 +113,12 @@ uniform vec4 iblData;
 #define isPbrTextureLoaded FSTexFlags.y > 0.0
 #define isIblTextureLoaded FSTexFlags.z > 0.0
 
+#define gameLightingMode gameLightingFlags.x
+#define GAME_LIGHTING_PER_PIXEL 2
+
 void main()
 {
-	vec4 color = vec4(toLinear(v_color0.rgb), v_color0.a);
+	vec4 color = v_color0;
```

Comment:
Can you please fix the indentation here to match the lines below?

Reply:
Fixed :)